### ISSUE TITLE
@storybook/addon-backgrounds does not work with static builds since v5

### ISSUE THREAD

**Issue by iernie** [storybooks/storybook#6213]
**Describe the bug**
`@storybook/addon-backgrounds` does not work with static builds since v5. It seems the register line in `addons.js` is the culprit. Without it, backgrounds doesn't work, but static build at least works.

**To Reproduce**
1. Start a brand new `create-react-app`
2. Initialize storybook and add `@storybook/addon-backgrounds` as per readme
3. Run normal build, see that backgrounds work
4. Run static build, see that the page crashes. The UI flashes for a second before going all white.
5. Run static build without  `import '@storybook/addon-backgrounds/register';` in `addons.js`, see that it now works (but without backgrounds).

**Expected behavior**
`@storybook/addon-backgrounds` should work in static builds.

**Code snippets**
These are the console errors I get when including backgrounds.

```
Uncaught TypeError: Cannot read property 'addEventListener' of null
    at HTMLIFrameElement.n (vendors~main.dc1a670af8326ca264f8.bundle.js:66)

TypeError: Cannot read property 'addEventListener' of null
    at n (vendors~main.dc1a670af8326ca264f8.bundle.js:66)
    at vendors~main.dc1a670af8326ca264f8.bundle.js:66
    at Array.forEach (<anonymous>)
    at t.componentDidMount (vendors~main.dc1a670af8326ca264f8.bundle.js:66)
    at Ha (vendors~main.dc1a670af8326ca264f8.bundle.js:74)
    at Wa (vendors~main.dc1a670af8326ca264f8.bundle.js:74)
    at vendors~main.dc1a670af8326ca264f8.bundle.js:74
    at Object.t.unstable_runWithPriority (vendors~main.dc1a670af8326ca264f8.bundle.js:83)
    at Mu (vendors~main.dc1a670af8326ca264f8.bundle.js:74)
    at Cu (vendors~main.dc1a670af8326ca264f8.bundle.js:74)

TypeError: Cannot read property 'addEventListener' of null
    at n (vendors~main.dc1a670af8326ca264f8.bundle.js:66)
    at vendors~main.dc1a670af8326ca264f8.bundle.js:66
    at Array.forEach (<anonymous>)
    at t.componentDidMount (vendors~main.dc1a670af8326ca264f8.bundle.js:66)
    at Ha (vendors~main.dc1a670af8326ca264f8.bundle.js:74)
    at Wa (vendors~main.dc1a670af8326ca264f8.bundle.js:74)
    at vendors~main.dc1a670af8326ca264f8.bundle.js:74
    at Object.t.unstable_runWithPriority (vendors~main.dc1a670af8326ca264f8.bundle.js:83)
    at Mu (vendors~main.dc1a670af8326ca264f8.bundle.js:74)
    at Cu (vendors~main.dc1a670af8326ca264f8.bundle.js:74)

vendors~main.dc1a670af8326ca264f8.bundle.js:6 TypeError: Cannot read property 'addEventListener' of null
    at n (vendors~main.dc1a670af8326ca264f8.bundle.js:66)
    at vendors~main.dc1a670af8326ca264f8.bundle.js:66
    at Array.forEach (<anonymous>)
    at t.componentDidMount (vendors~main.dc1a670af8326ca264f8.bundle.js:66)
    at Ha (vendors~main.dc1a670af8326ca264f8.bundle.js:74)
    at Wa (vendors~main.dc1a670af8326ca264f8.bundle.js:74)
    at vendors~main.dc1a670af8326ca264f8.bundle.js:74
    at Object.t.unstable_runWithPriority (vendors~main.dc1a670af8326ca264f8.bundle.js:83)
    at Mu (vendors~main.dc1a670af8326ca264f8.bundle.js:74)
    at Cu (vendors~main.dc1a670af8326ca264f8.bundle.js:74)
```

Note that the first error I also get when not including the backgrounds addon
```
Uncaught TypeError: Cannot read property 'addEventListener' of null
    at HTMLIFrameElement.n (vendors~main.89e1fda93dc1c7d7d1ad.bundle.js:66)
```
but this error at least doesn't seem to break/crash the page (this might be a separate issue)

**System:**
 - OS: Windows 10
 - Device: Desktop
 - Browser: Chrome
 - Framework: React
 - Addons: @storybook/addon-backgrounds
 - Version: 5.0.3

**Comment by shilman** [comment]
I tried this in `create-react-app` and was not able to repro. Here are my versions:

```
  "dependencies": {
    "react": "^16.8.4",
    "react-dom": "^16.8.4",
    "react-scripts": "2.1.8"
  },
  "devDependencies": {
    "@babel/core": "^7.3.4",
    "@storybook/addon-actions": "^5.0.3",
    "@storybook/addon-backgrounds": "^5.0.3",
    "@storybook/addon-links": "^5.0.3",
    "@storybook/addons": "^5.0.3",
    "@storybook/react": "^5.0.3",
    "babel-loader": "^8.0.5"
  }
```

Here is my addons.js:
```
import '@storybook/addon-actions/register';
import '@storybook/addon-links/register';
import '@storybook/addon-backgrounds/register';
```

And here is my config.js:

```js
import { configure, addParameters } from "@storybook/react";

addParameters({
  backgrounds: [
    { name: "twitter", value: "#00aced", default: true },
    { name: "facebook", value: "#3b5998" }
  ]
});

function loadStories() {
  require("../src/stories");
}

configure(loadStories, module);
```

**Comment by iernie** [comment]
Here is a repo with a project I am able to reproduce with.
https://github.com/iernie/storybook-background-issue

It crashes in Chrome 73.0.3683.86 (even incognito) and Edge. Interestingly enough it doesn't seem to crash in Firefox.
Node v11.12.0
npm v6.9.0

**Comment by Armanio** [comment]
I think it's already fixed in 5.1.0-alpha.13. 🤔

**Comment by iernie** [comment]
This seems to be fixed in `5.0.5`. Thanks!
